spec/spec_helper.rb

```diff
@@ -64,12 +64,12 @@
   # Many RSpec users commonly either run the entire suite or an individual
   # file, and it's useful to allow more verbose output when running an
   # individual spec file.
-  # if config.files_to_run.one?
+  if config.files_to_run.one?
     # Use the documentation formatter for detailed output,
     # unless a formatter has already been configured
     # (e.g. via a command-line flag).
-    # config.default_formatter = 'doc'
-  # end
+    config.default_formatter = 'doc'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.